pkg/apis/controlplane/v1beta1/conversion.go

```diff
@@ -107,3 +107,249 @@ func Convert_controlplane_GroupMember_To_v1beta1_GroupMember(in *controlplane.Gr
 	out.Endpoints = endpoints
 	return nil
 }
+
+func Convert_v1beta1_GroupMemberPod_To_controlplane_GroupMember(in *GroupMemberPod, out *controlplane.GroupMember, s conversion.Scope) error {
```

Comment:
out-of-curiosity, will a function like this one ever be called by the K8s apimachinery or is it only called by our own conversion functions (e.g. `Convert_v1beta1_AddressGroup_To_controlplane_AddressGroup`), given that `GroupMember` is not a "top-level" K8s resource?

Reply:
I believe only by out own conversion functions. Just wrote that for the sake of completeness.